src/rvsdg/tree_unique/to_tree.rs

```diff
@@ -135,8 +166,9 @@ impl<'a> RegionTranslator<'a> {
             BasicExpr::Print(args) => {
                 assert!(args.len() == 2, "print should have 2 arguments");
                 let arg1 = self.translate_operand(args[0]);
+                // should be unit
```

Comment:
What should be unit?

Reply:
Added a comment- argument 2 is the incoming print edge state, which we translate to unit in the tree encoding.